pom.xml

```diff
@@ -232,8 +232,8 @@
                             <systemPropertyVariables>
                                 <profile.id>${profile.id}</profile.id>
                                 <!-- S2i configuration for OpenShift -->
-                                <ts.global.s2i.quarkus.jvm.builder.image>${ts.global.s2i.quarkus.jvm.builder.image}</ts.global.s2i.quarkus.jvm.builder.image>
-                                <ts.global.s2i.quarkus.native.builder.image>${ts.global.s2i.quarkus.native.builder.image}</ts.global.s2i.quarkus.native.builder.image>
+                                <quarkus.s2i.base-jvm-image>${quarkus.s2i.base-jvm-image}</quarkus.s2i.base-jvm-image>
```

Comment:
For OpenShift, Quarkus docs say it uses OpenJDK 17 for 17+ and 11 for 11 builds https://quarkus.io/guides/all-config#quarkus-container-image-openshift_quarkus.s2i.base-jvm-image. Are you sure we really want to use 11 in all cases? IIRC until now when not specified, we were using default which is OpenJDK 17 from `io.quarkus.test.services.quarkus.model.QuarkusProperties#QUARKUS_JVM_S2I`.